FOCUS feature level introduction - mandatory / conditional / optional (#331)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/columns/listcost.md

# List Cost

List Cost represents the cost calculated by multiplying the [*list unit price*](#glossary:list-unit-price) and the corresponding [Pricing Quantity](#pricingquantity). List Cost is denominated in the [Billing Currency](#billingcurrency) and is commonly used for calculating savings based on various rate optimization activities, by comparing it with [Billed Cost](#billedcost) and [Effective Cost](#effectivecost).

> The ListCost column MUST be present in the billing data and MUST NOT be null. This column MUST be of type Decimal, MUST conform to [Numeric Format](#numericformat) requirements, and be denominated in the BillingCurrency. Multiplying the *list unit price* by PricingQuantity MUST produce the ListCost.

Reviewer: Do we refer to the glossary instead of directly mentioning the actual columns in the normative paragraph, only in the List Cost and ListUnitPrice specs? Taking into account that it is normative, I would prefer the reference to the actual column.
Consider something like this:

```suggestion
The ListCost column MUST be present in the billing data and MUST NOT be null. This column MUST be of type Decimal, MUST conform to [Numeric Format](#numericformat) requirements, and be denominated in the BillingCurrency. When [ListUnitPrice](#listunitprice) is present and is not null, multiplying the ListUnitPrice by PricingQuantity MUST produce the ListCost.
```

Author: I was debating this. I'm good with the suggestion. 

(previously, I was debating if we should refer to the glossary entry to remove the complication of list unit price being conditional)

In cases where the ListUnitPrice is present and is null, the following applies:

> * The ListCost MUST be calculated based on the ListCost of the related charges if the charge is calculated based on other charges (e.g. [ChargeCategory](#chargecategory) is "Tax").

Reviewer: ```suggestion
* The ListCost of a charge calculated based on other charges (e.g., when the [ChargeCategory](#chargecategory) is "Tax") MUST be calculated based on the ListCost of those related charges.
```

> * The ListCost MUST match the [BilledCost](#billedcost) if the charge is unrelated to other charges (e.g. [ChargeSubcategory](#chargesubcategory) is "Credit").

Reviewer: ```suggestion
* The ListCost of a charge unrelated to other charges (e.g., when the [ChargeSubcategory](#chargesubcategory) is "Credit") MUST match the [BilledCost](#billedcost).
```

| Constraint      | Value                   |
|:----------------|:------------------------|
| Column type     | Metric                  |

> | FOCUS Essential | True                    |

Reviewer 2: Does this require that the vendor publish a list cost...though that may not actually be published? I see this as a stumbling block for many vendors to adopt even single line item focus data.

Reviewer 2: Question answered in discussion today. @ijurica  perhaps we ensure it is clear that in the absence of a list price it matches BilledCost.

Author: We're not specifying that it needs to be 'public' anywhere. Many providers don't publish rates publicly but they will privately tell you the starting point for price/cost (list price/cost) and then go from there to a negotiated price/cost for you.

Author: "The List Unit Price represents the suggested provider-published unit price for a single Pricing Unit of the
associated SKU, exclusive of any discounts."

Reviewer 3: I interpreted "provider-published" to mean that I must publicly "publish" prices. If we remove that, the sentence still conveys the same point. Maybe we just simplify?

> _The List Unit Price represents the suggested unit price for a single Pricing Unit of the
associated SKU, exclusive of any discounts._

Author: > The List Unit Price represents the Provider suggested unit price for a single Pricing Unit of the
associated SKU, exclusive of any discounts.

How about this?

Reviewer 4: Created discussion topic #366 to continue identification of a resolution for this
